apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -0,0 +1,388 @@
+        const defaultColumns: TableColumnInterface[] = [
+            {
+                allowToggleVisibility: false,
+                dataIndex: "username",
+                id: "username",
+                key: "username",
+                render: (response: BulkUserImportOperationResponse): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-username-item-heading` }>
+                            <Header.Content>
+                                {
+                                    response.username
+                                }
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.username")
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "status",
+                id: "status",
+                key: "status",
+                render: (response: BulkUserImportOperationResponse): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}--status-item-heading` } >
+                            <Header.Content>
+                                <Label
+                                    data-componentid={ `${componentId}-bulk-label` }
+                                    content={ response.status }
+                                    size="small"
+                                    color={ response.statusCode === BulkUserImportStatus.SUCCESS ? "green" : "red" }
+                                    className={ "group-label" }
+                                />
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.status")
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "message",
+                id: "message",
+                key: "message",
+                render: (response: BulkUserImportOperationResponse): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-status-item-heading` }>
+                            <Header.Content>
+                                { response.message }
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.message")
+            }
+        ];
+
+        return defaultColumns;
+    };
+    
+    /**
+     * Filters the response list based on the search query and the selected status.
+     */
+    const filterResponseList = () => {
```

Comment:
Shall we improve the name?
```suggestion
    const filterBulkUsersResponseList = () => {
```